src/components/Cart/CartItems/CartItems.jsx

```diff
@@ -31,6 +41,49 @@ export const CartItems = () => {
     }
   };
 
+  const inWishlist = (_id) => {
+    const productInWishlist = wishlistState.wishlist.find(
+      (product) => product._id === _id
+    );
+
+    return productInWishlist ? true : false;
+  };
+
+  const listProducts = async () => {
+    try {
+      const response = await axios.get("/api/products");
+      if (response.status === 200) {
+        setProducts(response.data.products);
+      } else {
+        throw new Error();
+      }
+    } catch (error) {
+      console.log(error);
```

Comment:
Instead of consoling the error you can use alert here

Reply:
Done